### PR TITLE
Checker message examples

### PR DESCRIPTION
**Pull request by nigef**
Added some text to the docstring that will eventually go into the `index.md`

### CODE REVIEW

File: examples/pylint/R0901_too_many_ancestors.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+"""pylint: too many ancestors
```
Comment (david-yz-liu) [inline examples/pylint/R0901_too_many_ancestors.py]: Disable this check

File: examples/pylint/R0902_too_many_instance_attributes.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+"""pylint: too many instance attributes
```
Comment (david-yz-liu) [inline examples/pylint/R0902_too_many_instance_attributes.py]: If you have to use class attributes to do this check, disable it.

File: examples/pylint/R0903_too_few_public_methods.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+"""pylint: too few public methods
```
Comment (david-yz-liu) [inline examples/pylint/R0903_too_few_public_methods.py]: Disable this check

File: examples/pylint/R0904_too_many_public_methods.py [hunk 4]
```diff
@@ -0,0 +1,94 @@
+"""pylint: too many public methods
```
Comment (david-yz-liu) [inline examples/pylint/R0904_too_many_public_methods.py]: Disable this check

File: examples/pylint/R0911_too_many_return_statements.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
+"""pylint: too many return statements
```
Comment (david-yz-liu) [inline examples/pylint/R0911_too_many_return_statements.py]: Disable this check

File: examples/pylint/R0912_too_many_branches.py [hunk 6]
```diff
@@ -0,0 +1,87 @@
+"""pylint: too many branches
+
+Used when a function or method has too many branches, making it hard to follow.
+
+Default: 12
+"""
+
+def fun1():
```
Comment (david-yz-liu) [inline examples/pylint/R0912_too_many_branches.py]: I'm not really sure why these functions are in this file.

File: examples/pylint/R0913_too_many_arguments.py [hunk 7]
```diff
@@ -0,0 +1,7 @@
+"""pylint: too many arguments
+
+Default: 5
+"""
+
+def foo_bar(arg1, arg2, arg3, arg4, arg5, arg6):
+    """I have too many arguments"""
```
Comment (david-yz-liu) [inline examples/pylint/R0913_too_many_arguments.py]: Put a `pass`

File: examples/pylint/R0915_too_many_statements.py [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+    statement_39 = 39
+    statement_40 = 40
+    statement("function calls are statements too")
+    statement("function calls are statements too")
+    statement("function calls are statements too")
+    statement("function calls are statements too")
+    statement("function calls are statements too")
+    statement("function calls are statements too")
+    statement("function calls are statements too")
+    statment_also = "this is..." if arg else "...another statement"
```
Comment (david-yz-liu) [inline examples/pylint/R0915_too_many_statements.py]: Don't use the ternary if.

File: examples/pylint/W1501_bad_open_mode.py [hunk 9]
```diff
@@ -0,0 +1,14 @@
+"""pylint: bad open mode
```
Comment (david-yz-liu) [inline examples/pylint/W1501_bad_open_mode.py]: Disable this check. We will likely not want students using `open`

File: examples/pylint/R0902_too_many_instance_attributes.py (hunk 2 above)
Comment (nigef) [inline examples/pylint/R0902_too_many_instance_attributes.py]: Actually good point, I can do it without class attributes.

**Comment by david-yz-liu** [comment]
Thanks, @nigef!
